From 2674933e0e9ee1f22f40f148bea23f7c5c10512d Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Wed, 16 Nov 2022 13:05:08 +0530
Subject: [PATCH 0130/1141] arm64: dts: st: add USB2PHY1 and USB2PHY2 nodes in
 stm32mp257f-ev

Add the required node(s) for USB2PHY1 and USB2PHY2 support
These are 2 instances of same IP USB2-FEMTO-PHY used by
USB2H and USB3DRD ctrl respectively

Change-Id: I4f1bae7c1a2390849f83e9547afc9d71d172e4b5
Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305765
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
index d2e7fae93670..4cbc1455959d 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev.dts
@@ -96,3 +96,11 @@ &usart2 {
 	/delete-property/dma-names;
 	status = "okay";
 };
+
+&usb2_phy1 {
+	status = "okay";
+};
+
+&usb2_phy2 {
+	status = "okay";
+};
-- 
2.39.2

